help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,160 @@
+Custom uploads|X|Audiences [[imported]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/ui/overview#import-audience) into Experience Platform from CSV files.
+
+{style="table-layout:auto"}
+
+## Export type and frequency
+
+Refer to the table below for information about the destination export type and frequency.
+
+| **Item** | **Type** | **Notes**|
+|---|---|---|  
+|Export type|**Profile-based**|You are exporting all members of a segment, together with the desired schema fields (for example: email address, phone number, last name), as chosen in the select profile attributes screen of the [[destination activation workflow]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations#select-attributes).|
+|Frequency|**Batch**|Batch destinations export files to downstream platforms in increments of three, six, eight, twelve, or twenty-four hours. Read more about [[batch file-based frequency destinations]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-types#file-based).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination
+
+[!IMPORTANT]
+To connect to the destination, you need the **View Destinations** and **Manage and Activate Dataset Destinations** [[access control permissions]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions). Read the [[access control overview]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### New Account
+
+#### Authenticate to destination
+
+To authenticate to the destination, fill in the required fields and select **Connect to destination**.
+
+To access your bucket on Experience Platform, you need to provide valid values for the following credentials:
+
+
+| **Credential** | **Description** |
+|---|---|
+|Access key|The access key ID for your bucket. You can retrieve this value from the Merkury team.|
+|Secret key|The secret key ID for your bucket. You can retrieve this value from the Merkury team.|
+|Bucket name|This is your bucket where files will be shared. You can retrieve this value from the Merkury team.|
+
+{style="table-layout:auto"}
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image4.png)
```

Comment:
again, the alt-text is not helpful. based on the examples provided further above, please update the alt-text